### PR TITLE
Added more URLs

### PR DESCRIPTION
**Pull request by ShivamGoyal1899**
These are the URLs I've added
```
'*://*.pramp.com/*',
'*://uxdesign.cc/*',
'*://medium.muz.li/*',
'*://*.usejournal.com/*',
'*://timeline.com/*',
'*://substance.media/*',
'*://thebolditalic.com/*',
'*://thecreative.cafe/*',
'*://theascent.pub/*',
'*://ceoplaybook.io/*',
'*://dayoneperspective.com/*',
'*://eand.co/*',
'*://extranewsfeed.com/*',
'*://democracyguardian.com/*',
'*://*.issuevoter.org/*',
```

### CODE REVIEW

**Comment by ShivamGoyal1899** [comment]
@swapagarwal Can you please have a look?

**Comment by swapagarwal** [comment]
@ShivamGoyal1899 Taking a look. By the way, where did you get these from?

**Comment by ShivamGoyal1899** [comment]
@swapagarwal I just used a query in **Developer Tools > Console** to scrap out the URLs.  
Here's the query I used `n=$$('a');for(u in n)console.log(n[u].href)`

**Comment by ShivamGoyal1899** [comment]
@swapagarwal I've fixed a small typo with one domain. It was misspelled as **enad.co**

File: chrome/background.js [hunk 1]
```diff
@@ -47,6 +47,21 @@ chrome.webRequest.onBeforeSendHeaders.addListener(
...
       '*://bullshit.ist/*',
+      '*://*.pramp.com/*',
+      '*://uxdesign.cc/*',
+      '*://medium.muz.li/*',
+      '*://*.usejournal.com/*',
+      '*://timeline.com/*',
+      '*://substance.media/*',
+      '*://thebolditalic.com/*',
+      '*://thecreative.cafe/*',
+      '*://theascent.pub/*',
```
Comment (swapagarwal) [inline chrome/background.js]: Redirects to `medium.com/the-ascent`

**Comment by swapagarwal** [comment]
Thanks @ShivamGoyal1899! 👍

**Pull request merged**